examples/energy-management-app/energy-management-common/src/EVSEManufacturerImpl.cpp

```diff
@@ -1,6 +1,6 @@
 /*
  *
- *    Copyright (c) 2023 Project CHIP Authors
+ *    Copyright (c) 2024 Project CHIP Authors
```

Comment:
Still needs to not lose the old date.

Reply:
Changed to 2023-2024 as changes made in 2023 and 2024.